e2e-tests/mongod-major-upgrade-sharded/run

```diff
@@ -6,24 +6,6 @@ set -o xtrace
 test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 
-function get_mongod_ver_from_image() {
```

Comment:
If I'm not mistaking we use this function in `mongod_major_upgrade` test so let's remove it there as well.

Reply:
ok